tests/test_uploadanddownload.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import os
+import tempfile
+import subprocess
+
+from cacophonyapi.user import UserAPI
+
+class TestUploadDownload:
+    def test_upload(self, test_config):
+        os.chdir('../')
+        file_names = os.listdir('tests/test_data')
+        for file_name in file_names:
+            result = subprocess.run(
+                    ['python3',
+                    'cptv-upload.py',
+                    test_config.api_url,
+                    test_config.admin_username,
+                    test_config.admin_password,
+                    "test-group",
+                    "test-device",
+                    'tests/test_data/{0}'.format(file_name)]
+                )
+            assert result.returncode == 0
+
+    
+    def test_download(self, test_config):
+        if os.path.exists('downloads1'):
+            os.system('rm -r downloads1')
+
+        os.mkdir('downloads1')
+        result = subprocess.run(
+                    [
+                        'python3',
+                        'cptv-download.py',
+                        '-i',
+                        'None',
+                        '-m',
+                        'any',
+                        '-s',
+                        test_config.api_url,
+                        "downloads1",
+                        test_config.admin_username,
+                        test_config.admin_password
+                    ]
+                )
+
+        os.system('rm -r downloads1')
```

Comment:
use `os.rmdir`

Reply:
would be nice to have an assert that there is how ever many files that are expected in the downloads1 folder

Reply:
os.rmdir won't let me remove non-empty directories, so i used shutil.rmtree instead. as far as asserting the number of downloads is the same as the number of uploads: if the test is run more than once, this will fail because new entries are added to the database each time something is uploaded, even if it's the same file. For example, after running the test a few times i get: 
```FAILED test_uploadanddownload.py::TestUploadDownload::test_download - AssertionError: assert (0 == 0 and 4 == 39)```